pkgs/by-name/sy/sysvol/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  git,
+  gtk4-layer-shell,
+  gtkmm4,
+  nix-update-script,
+  pkg-config,
+  wireplumber,
+  wrapGAppsHook4,
+}:
+
+stdenv.mkDerivation {
+  pname = "sysvol";
+  version = "0-unstable-2024-06-07";
+
+  src = fetchFromGitHub {
+    owner = "System64fumo";
+    repo = "sysvol";
+    rev = "56d7dcda4b246e71b2c6d29cbb2315bddf446032";
+    hash = "sha256-Kz1Vsfh882gaGj1gqeT7HEQ/YJ1fK1zynpVWwD/YqzE=";
+    leaveDotGit = true;  # for `sysvol -v` to know its version
```

Comment:
This reduces reproducibility. I would recommend to just hardcode the information we already have or to just mock them.